README.md

```diff
@@ -405,6 +405,7 @@ Variable | Description | Default value
 `JVB_TCP_PORT` | TCP port for media used by Jitsi Videobridge when the TCP Harvester is enabled | 4443
 `JVB_BREWERY_MUC` | MUC name for the JVB pool | jvbbrewery
 `JVB_ENABLE_APIS` | Comma separated list of JVB APIs to enable | none
+`JIGASI_DEFAULT_ROOM_NAME` | Default room name to connect incoming calls to if no room header specified | siptest
```

Comment:
This change needs to be applied to https://github.com/jitsi/handbook/blob/master/docs/devops-guide/docker.md (cf: #601)